test/test_torch.py

```diff
@@ -3132,7 +3136,7 @@ def torchfn(t1, t2, t3):
                     elif fn == "masked_select":
                         return fntorch(t1, t2 < 0)
                     elif fn == "masked_scatter":
-                        return fntorch(t1, t2 < 0.5, cast(torch.arange(1, t1.nelement() + 1).float()))
+                        return fntorch(t1, t2 < 0.5, full1d)
```

Comment:
Since we are adding broadcasting to the first argument, having a vector of size `t1.nelement() + 1` is no longer enough.

Reply:
What do you mean by not enough?